src/back/auth/src/service/service.cpp

```diff
@@ -252,4 +256,38 @@ void Service::updateTokens(model::Session& session) const
 	session._idToken = tokens._idToken;
 }
 
+model::UserInfo Service::GetUserInfoById(const std::string& id, const std::string& sessionId)
+{
+	// Получаем сессию из базы
+	auto session = _session.Table().Get(sessionId);
+
+	// Обновляем OIDC токены если требуется
+	if (_tokenizer.IsExpired(session._accessToken))
+	{
+		// TODO: dest lock!
+		updateTokens(session);
+		_session.Table().UpdateTokens(session);
+	}
+
+	// Запрашиваем у OIDC инфу о пользователе
+	return _oidc.GetUserInfoById(id, session._accessToken);
+}
+
+std::vector<model::UserInfo> Service::GetUserInfoList(const std::string& search, const std::string& sessionId, unsigned int start, unsigned int limit) {
```

Comment:
Вместо unsigned int давай использовать uint32_t 
Извини, пропустил этот момент в space. Давай там тоже поправим. Могу отдельную задачу завести или сразу в этой же.
```suggestion
std::vector<model::UserInfo> Service::GetUserInfoList(const std::string& search, const std::string& sessionId, uint32_t start, uint32_t limit) {
```

Reply:
Готово. Везде поправлю в рамках этой задачи.